index.md

```diff
@@ -24,10 +24,13 @@ See the [official Anbox Cloud website](https://anbox-cloud.io/) for more informa
 
 ## What's new
 
-Along with bug fixes and general improvements, Anbox Cloud 1.13 comes with:
+Along with bug fixes and general improvements, Anbox Cloud 1.14 comes with:
```

Comment:
```suggestion
Along with bug fixes and general improvements, Anbox Cloud 1.14 includes the following changes:
```
Otherwise the sentence structure below doesn't work ;)